std/experimental/checkedint.d

```diff
@@ -351,12 +351,14 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
     /**
     Assignment operator. Has the same constraints as the constructor.
     */
-    void opAssign(U)(U rhs) if (is(typeof(Checked!(T, Hook)(rhs))))
+    Checked opAssign(U)(U rhs) return
```

Comment:
Chainable assignment operators should return by `ref`, not value:
```D
import std.stdio;

void changeIt(ref long it) pure @safe nothrow @nogc {
    it += 7;
}

void main() {
	long a, b;
    changeIt(a = b = 4);
    writeln(a); // prints 11
}
```

Reply:
```suggestion
    ref Checked opAssign(U)(U rhs) return
```

Reply:
Done, thanks!